subliminal/providers/tvsubtitles.py

```diff
@@ -105,15 +108,35 @@ def find_show_id(self, series):
         logger.debug('Searching series %r', data)
         soup = self.request('/search.php', data=data, method='POST')
         links = soup.select('div.left li div a[href^="/tvshow-"]')
+        _series = IGNORE_DATEMATCH.match(
+            IGNORED_CHARACTERS_RE.sub('', series)\
+                .replace('.', ' ').strip().lower(),
+        )
+        if not _series:
+            _series = IGNORED_CHARACTERS_RE.sub('', series)\
+                        .replace('.', ' ').strip().lower()
+        else:
+            _series = _series.group(1)
+
```

Comment:
A little comment on this and an explicit name instead of `_series` would be nice :star2: